package.json

```diff
@@ -132,21 +132,21 @@
         "ws": "1.0.1"
     },
     "devDependencies": {
-        "del": "2.2.0",
-        "gulp": "^3.9.0",
+        "del": "^2.2.0",
+        "gulp": "^3.9.1",
         "gulp-mocha": "^2.2.0",
-        "gulp-sourcemaps": "^1.5.2",
-        "gulp-tslint": "^3.3.1",
-        "gulp-typescript": "^2.8.0",
-        "gulp-util": "^3.0.5",
+        "gulp-sourcemaps": "^1.6.0",
+        "gulp-tslint": "^4.3.3",
+        "gulp-typescript": "^2.12.1",
+        "gulp-util": "^3.0.7",
         "mocha-teamcity-reporter": "^1.0.0",
         "run-sequence": "^1.1.5",
         "sinon": "^1.17.3",
-        "should": "^8.2.2",
+        "should": "^8.3.0",
         "through2": "^2.0.1",
-        "tslint": "^2.5.1",
-        "typescript": "^1.7.5",
-        "vsce": "1.0.0",
+        "tslint": "^3.6.0",
+        "typescript": "^1.8.9",
+        "vsce": "^1.3.0",
         "vscode": "^0.10.7"
```

Comment:
What is the relationship between the `vscode` package version and the VSCode IDE version? What I am trying to figure out is how often should we update this package, or if it should be in sync somehow with the VSCode versions we support.